src/components/User/User.js

```diff
@@ -1,8 +1,11 @@
+import React, { useState } from 'react'
 import './User.css';
 
 const User = ({ user }) => {
   const { about, age, company, country, name, photo } = user;
 
+  const [expanded, setExpanded ] = useState(false)
```

Comment:
You're on the right track here, too. This implementation works for us but in order to implement Expand All / Collapse All, it needs to be lifted to the parent